src/domain/Phone/WebRTCPhone.js

```diff
@@ -542,6 +563,8 @@ export default class WebRTCPhone extends Emitter implements Phone {
   }
 
   unhold(callSession: CallSession, withEvent: boolean = true): void {
+    IssueReporter.log(IssueReporter.INFO, '[WebRtcPhone] unhold', callSession.getId());
```

Comment:
`callSession ? callSession.getId() : null`? I'm only bringing this up because that's how you've handled it a few lines below; at your own discretion